server/src/controllers/article.controller.js

```diff
@@ -0,0 +1,111 @@
+import models from '../database/models';
+
+const { Articles } = models;
+/**
+   * create a new article
+   * @method createArticle
+   * @param {object} req
+   * @param {object} res
+   * @returns {(function|object)} Function next() or JSON object
+   */
+export const createArticle = async (req, res) => {
+  const { id: userId } = req.decoded;
+
+  const data = await Articles.create({ ...req.body, userId });
```

Comment:
Please use descriptive variable names. In this case, "article" would work just fine